tm2py/components/network/highway/highway_assign.py

```diff
@@ -254,7 +254,7 @@ def _calc_weighted_avg_flow(self, scenario: EmmeScenario):
         write_iteration_flow = self.controller.config.highway.msa.write_iteration_flow
         net_calc = NetworkCalculator(scenario)
 
-        if iteration == 1: 
+        if iteration == 1 or self.controller.config.run.warmstart: 
```

Comment:
```suggestion
        if iteration == 1 or ( (iteration == 0) & self.controller.config.run.warmstart): 
```
otherwise the elif will never be triggered.